CHANGES.rst

```diff
@@ -25,22 +25,22 @@ Release History
 **Added**
 
 - Added progress bar support for Jupyter Lab >=0.32.
-  (`#1428 <https://github.com/nengo/nengo/pull/1428>`__,
-  `#1087 <https://github.com/nengo/nengo/issues/1087>`__)
+  (`#1428 <https://github.com/nengo/nengo/pull/1428>`_,
+  `#1087 <https://github.com/nengo/nengo/issues/1087>`_)
```

Comment:
So this will cause ReST parsing issues whenever we have a hyperlink that is used multiple times. The `_` gives you a link that is associated with the text, while `__` gives an "anonymous" link (see [here](http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup)). If you use a link with `_` twice with different text you get a parse error.

So, if we wanted to be fully consistent we should use `__` everywhere, which we've sort of started to do since we ran into that problem in the past. However, I really think we should be touching these old lines because they don't cause any parse errors, and it messed up the `git blame` if you want to look at the changelog and see when something was added.

Reply:
Fair enough (resolved).